main/py3-meld3/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Peter Bui <pnutzh4x0r@gmail.com>
+# Maintainer:
+pkgname=py3-meld3
+_pkgname=meld3
+pkgver=1.0.2
+pkgrel=2
+pkgdesc="An HTML/XML templating engine"
+url="https://github.com/supervisor/meld3"
+arch="noarch"
+license="Repoze"
```

Comment:
Repoze is not in the license list as far as I can tell. Switch to custom?

Reply:
sounds like it should be custom

Reply:
Seems like it's just a plain old `BSD-4-Clause`, no?

Reply:
Seems different. See https://directory.fsf.org/wiki/License:BSD-4-Clause . Especially point 4.